sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -1056,4 +1056,8 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 #IfNotRow4D supported_external_dataloads load_type ICD10 load_source CMS load_release_date 2021-10-01 load_filename Zip File 3 2022 ICD-10-PCS Codes File.zip
 INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_release_date`, `load_filename`, `load_checksum`) VALUES
 ('ICD10', 'CMS', '2021-10-01', 'Zip File 3 2022 ICD-10-PCS Codes File.zip', 'a432177acbdaf9908aa528078ae72176');
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfMissingColumn audit_master is_qrda_document
+ALTER TABLE `audit_master` ADD `is_qrda_document` BOOLEAN NULL DEFAULT FALSE;
```

Comment:
By decree, today on September 28th, 2021 is the day that the first boolean column was placed in OpenEMR, so this day will be declared OpenEMR International Boolean Day! :)

![2DV-2](https://user-images.githubusercontent.com/278968/135199253-ebc4b370-f852-41f2-80fe-6596d995ae4f.gif)

Reply:
Lol, I've been wondering why we've been avoiding boolean.  I figured it was because we wanted to deal with db compatibility.

Reply:
Is there a compatibility issue with boolean I'm unaware?
I also brought in the first and only view to Sherwins chagrin.:)

Reply:
boolean should be just fine (looks like mysql/mariadb under the hood basically use tinyint(1) for this)